src/Umbraco.Web.UI.Client/src/common/services/keyboard.service.js

```diff
@@ -264,7 +264,7 @@ function keyboardService($window, $timeout) {
 
         var elt;
         // Initialize opt object
-        opt   = angular.extend({}, defaultOpt, opt);
+        opt   = Utilities.extend(defaultOpt, opt);
```

Comment:
@ronaldbarendse the `angular.extend({}, object1, object2)` allowed a second it more parameters (source objects) to merged with the empty object (however note angular.extend does not support recursive merge (deep copy)): angular.extend does not support recursive merge (deep copy).

So if we just used `Utilities.extend({}, defaultOpt, opt);` it would return an empty object, because the `Utilities.extend` only use `angular.extend` with two arguments.

Reply:
This still needs to be updated to `opt = Utilities.extend({}, defaultOpt, opt);`, as it would otherwise merge the current options (`opt`) into the default options (`defaultOpt`) and cause lots of issues when used multiple times...

Reply:
I recall I just logged the `opt` variable before and after here, but not sure if it would have other side effects as you mention.

Reply:
After merging this PR, it's now fixed in https://github.com/umbraco/Umbraco-CMS/pull/10348 👍🏻